### DISCUSSION TITLE
dashboard colorscript issue

### DISCUSSION THREAD

**Discussion by oaflindberg** [folke/snacks.nvim#226] [Q&A]
Might be a really dumb question, but I can't seem to get colorscript to work. I tried installing EntySec/ColorScript, but it wouldn't work at all. Maybe there's another colorscript that I couldn't find?

Any advice at all would be much appreciated 🙏🏼

**Comment by folke** [comment]
What are you talking about? Wrong repo?

**Reply by oaflindberg** [reply]
Oh, sorry. I should've been more clear. When trying to use the advanced example of the dashboard (the one in docs/dashboard.md), that uses cmd = "colorscript -e square". That's what I can't get to work

**Reply by folke** [reply]
https://github.com/charitarthchugh/shell-color-scripts

**Reply by bro69noscope** [reply]
`EntySec/ColorScript` shows up first in Google results searching for `colorscript "-e" square`
